exporter/storage/storage.go

```diff
@@ -29,6 +31,11 @@ func NewExporterStorage(db basedb.IDb, logger *zap.Logger) Storage {
 	return &es
 }
 
+// Clean clears all information
+func (es *exporterStorage) Clean() error {
```

Comment:
@amir-blox this deletes the entire storage?

Reply:
It will delete exporter storage - validators/operators information, which is saved in a different collection than shares